ERROR - 2023-01-23 01:12:07 --> mysqli_sql_exception: Unknown column 'attached_file_name' in 'field list' in C:\xampp\htdocs\sumscircle\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\sumscircle\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `pr...', 0)
#1 C:\xampp\htdocs\sumscircle\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pr...')
#2 C:\xampp\htdocs\sumscircle\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pr...')
#3 C:\xampp\htdocs\sumscircle\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pr...', Array, false)
#4 C:\xampp\htdocs\sumscircle\app\Models\Catlog_model.php(336): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\sumscircle\app\Controllers\Catlog.php(149): App\Models\Catlog_model->add_product(Array, Array)
#6 C:\xampp\htdocs\sumscircle\system\CodeIgniter.php(927): App\Controllers\Catlog->add_products()
#7 C:\xampp\htdocs\sumscircle\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Catlog))
#8 C:\xampp\htdocs\sumscircle\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\sumscircle\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-01-23 01:12:07 --> Unknown column 'attached_file_name' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 292.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `product_attatchedfile_t` (`product_id`, `attached_file_name`) VALUES (10, \'project1.jpeg\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `product_attatchedfile_t` (`product_id`, `attached_file_name`) VALUES (10, \'project1.jpeg\')')
 3 SYSTEMPATH\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `product_attatchedfile_t` (`product_id`, `attached_file_name`) VALUES (10, \'project1.jpeg\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `product_attatchedfile_t` (`product_id`, `attached_file_name`) VALUES (:product_id:, :attached_file_name:)', [...], false)
 5 APPPATH\Models\Catlog_model.php(336): CodeIgniter\Database\BaseBuilder->insert([...])
 6 APPPATH\Controllers\Catlog.php(149): App\Models\Catlog_model->add_product([...], [...])
 7 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Catlog->add_products()
 8 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Catlog))
 9 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-01-23 01:36:16 --> Cannot redeclare App\Models\Catlog_model::getproduct_variation()
in APPPATH\Models\Catlog_model.php on line 396.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2023-01-23 01:37:14 --> Undefined array key "attached_file_name"
in APPPATH\Views\Catlog\edit_products.php on line 269.
 1 APPPATH\Views\Catlog\edit_products.php(269): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "attached_file_name"', 'APPPATH\\Views\\Catlog\\edit_products.php', 269)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\Catlog\\edit_products.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1134): CodeIgniter\View\View->render('Catlog/edit_products', [], true)
 5 APPPATH\Controllers\Catlog.php(180): view('Catlog/edit_products', [...])
 6 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Catlog->edit_product('2')
 7 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Catlog))
 8 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-01-23 02:02:41 --> Too few arguments to function App\Models\Catlog_model::update_product(), 2 passed in C:\xampp\htdocs\sumscircle\app\Controllers\Catlog.php on line 211 and exactly 3 expected
in APPPATH\Models\Catlog_model.php on line 396.
 1 APPPATH\Controllers\Catlog.php(211): App\Models\Catlog_model->update_product([...], [...])
 2 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Catlog->update_product()
 3 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Catlog))
 4 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-01-23 02:09:35 --> mysqli_sql_exception: Unknown column 'attached_file_name' in 'field list' in C:\xampp\htdocs\sumscircle\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\sumscircle\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `pr...', 0)
#1 C:\xampp\htdocs\sumscircle\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pr...')
#2 C:\xampp\htdocs\sumscircle\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pr...')
#3 C:\xampp\htdocs\sumscircle\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pr...', Array, false)
#4 C:\xampp\htdocs\sumscircle\app\Models\Catlog_model.php(441): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\xampp\htdocs\sumscircle\app\Controllers\Catlog.php(211): App\Models\Catlog_model->update_product(Array, Array)
#6 C:\xampp\htdocs\sumscircle\system\CodeIgniter.php(927): App\Controllers\Catlog->update_product()
#7 C:\xampp\htdocs\sumscircle\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Catlog))
#8 C:\xampp\htdocs\sumscircle\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\sumscircle\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-01-23 02:09:35 --> Unknown column 'attached_file_name' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 292.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `product_attatchedfile_t` (`product_id`, `attached_file_name`) VALUES (\'1\', \'collection8.png\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `product_attatchedfile_t` (`product_id`, `attached_file_name`) VALUES (\'1\', \'collection8.png\')')
 3 SYSTEMPATH\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `product_attatchedfile_t` (`product_id`, `attached_file_name`) VALUES (\'1\', \'collection8.png\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `product_attatchedfile_t` (`product_id`, `attached_file_name`) VALUES (:product_id:, :attached_file_name:)', [...], false)
 5 APPPATH\Models\Catlog_model.php(441): CodeIgniter\Database\BaseBuilder->insert([...])
 6 APPPATH\Controllers\Catlog.php(211): App\Models\Catlog_model->update_product([...], [...])
 7 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Catlog->update_product()
 8 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Catlog))
 9 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-01-23 02:58:29 --> syntax error, unexpected token ")"
in APPPATH\Models\Catlog_model.php on line 475.
 1 SYSTEMPATH\Autoloader\Autoloader.php(259): CodeIgniter\Autoloader\Autoloader->includeFile('APPPATH\\Models\\Catlog_model.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(237): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Models\\Catlog_model')
 3 APPPATH\Controllers\Catlog.php(12): CodeIgniter\Autoloader\Autoloader->loadClass('App\\Models\\Catlog_model')
 4 SYSTEMPATH\CodeIgniter.php(889): App\Controllers\Catlog->__construct()
 5 SYSTEMPATH\CodeIgniter.php(473): CodeIgniter\CodeIgniter->createController()
 6 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-01-23 03:00:14 --> Undefined array key "hiddenimg"
in APPPATH\Models\Catlog_model.php on line 402.
 1 APPPATH\Models\Catlog_model.php(402): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "hiddenimg"', 'APPPATH\\Models\\Catlog_model.php', 402)
 2 APPPATH\Controllers\Catlog.php(211): App\Models\Catlog_model->update_product([...], [...])
 3 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Catlog->update_product()
 4 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Catlog))
 5 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-01-23 03:24:05 --> mysqli_sql_exception: Unknown column 'attached_file_name' in 'field list' in C:\xampp\htdocs\sumscircle\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\sumscircle\system\Database\MySQLi\Connection.php(292): mysqli->query('SELECT attached...', 0)
#1 C:\xampp\htdocs\sumscircle\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('SELECT attached...')
#2 C:\xampp\htdocs\sumscircle\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT attached...')
#3 C:\xampp\htdocs\sumscircle\app\Models\Catlog_model.php(511): CodeIgniter\Database\BaseConnection->query('SELECT attached...')
#4 C:\xampp\htdocs\sumscircle\app\Controllers\Catlog.php(186): App\Models\Catlog_model->delete_img('13')
#5 C:\xampp\htdocs\sumscircle\system\CodeIgniter.php(927): App\Controllers\Catlog->delete_img()
#6 C:\xampp\htdocs\sumscircle\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Catlog))
#7 C:\xampp\htdocs\sumscircle\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\sumscircle\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2023-01-23 03:24:05 --> Unknown column 'attached_file_name' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 292.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(292): mysqli->query('SELECT attached_file_name  FROM `product_attatchedfile_t` WHERE id = \'13\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('SELECT attached_file_name  FROM `product_attatchedfile_t` WHERE id = \'13\'')
 3 SYSTEMPATH\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT attached_file_name  FROM `product_attatchedfile_t` WHERE id = \'13\'')
 4 APPPATH\Models\Catlog_model.php(511): CodeIgniter\Database\BaseConnection->query('SELECT attached_file_name  FROM `product_attatchedfile_t` WHERE id = \'13\'')
 5 APPPATH\Controllers\Catlog.php(186): App\Models\Catlog_model->delete_img('13')
 6 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Catlog->delete_img()
 7 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Catlog))
 8 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-01-23 03:24:27 --> mysqli_sql_exception: Unknown column 'attached_file_name' in 'field list' in C:\xampp\htdocs\sumscircle\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\sumscircle\system\Database\MySQLi\Connection.php(292): mysqli->query('SELECT attached...', 0)
#1 C:\xampp\htdocs\sumscircle\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('SELECT attached...')
#2 C:\xampp\htdocs\sumscircle\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT attached...')
#3 C:\xampp\htdocs\sumscircle\app\Models\Catlog_model.php(511): CodeIgniter\Database\BaseConnection->query('SELECT attached...')
#4 C:\xampp\htdocs\sumscircle\app\Controllers\Catlog.php(186): App\Models\Catlog_model->delete_img('13')
#5 C:\xampp\htdocs\sumscircle\system\CodeIgniter.php(927): App\Controllers\Catlog->delete_img()
#6 C:\xampp\htdocs\sumscircle\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Catlog))
#7 C:\xampp\htdocs\sumscircle\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\sumscircle\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2023-01-23 03:24:27 --> Unknown column 'attached_file_name' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 292.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(292): mysqli->query('SELECT attached_file_name  FROM `product_attatchedfile_t` WHERE id = \'13\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('SELECT attached_file_name  FROM `product_attatchedfile_t` WHERE id = \'13\'')
 3 SYSTEMPATH\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT attached_file_name  FROM `product_attatchedfile_t` WHERE id = \'13\'')
 4 APPPATH\Models\Catlog_model.php(511): CodeIgniter\Database\BaseConnection->query('SELECT attached_file_name  FROM `product_attatchedfile_t` WHERE id = \'13\'')
 5 APPPATH\Controllers\Catlog.php(186): App\Models\Catlog_model->delete_img('13')
 6 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Catlog->delete_img()
 7 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Catlog))
 8 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-01-23 03:26:39 --> mysqli_sql_exception: Unknown column 'attached_file_name' in 'field list' in C:\xampp\htdocs\sumscircle\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\sumscircle\system\Database\MySQLi\Connection.php(292): mysqli->query('SELECT attached...', 0)
#1 C:\xampp\htdocs\sumscircle\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('SELECT attached...')
#2 C:\xampp\htdocs\sumscircle\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT attached...')
#3 C:\xampp\htdocs\sumscircle\app\Models\Catlog_model.php(511): CodeIgniter\Database\BaseConnection->query('SELECT attached...')
#4 C:\xampp\htdocs\sumscircle\app\Controllers\Catlog.php(186): App\Models\Catlog_model->delete_img('13')
#5 C:\xampp\htdocs\sumscircle\system\CodeIgniter.php(927): App\Controllers\Catlog->delete_img()
#6 C:\xampp\htdocs\sumscircle\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Catlog))
#7 C:\xampp\htdocs\sumscircle\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\sumscircle\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2023-01-23 03:26:39 --> Unknown column 'attached_file_name' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 292.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(292): mysqli->query('SELECT attached_file_name  FROM `product_attatchedfile_t` WHERE id = \'13\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('SELECT attached_file_name  FROM `product_attatchedfile_t` WHERE id = \'13\'')
 3 SYSTEMPATH\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT attached_file_name  FROM `product_attatchedfile_t` WHERE id = \'13\'')
 4 APPPATH\Models\Catlog_model.php(511): CodeIgniter\Database\BaseConnection->query('SELECT attached_file_name  FROM `product_attatchedfile_t` WHERE id = \'13\'')
 5 APPPATH\Controllers\Catlog.php(186): App\Models\Catlog_model->delete_img('13')
 6 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Catlog->delete_img()
 7 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Catlog))
 8 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-01-23 03:28:11 --> Undefined array key "attached_file_name"
in APPPATH\Models\Catlog_model.php on line 514.
 1 APPPATH\Models\Catlog_model.php(514): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "attached_file_name"', 'APPPATH\\Models\\Catlog_model.php', 514)
 2 APPPATH\Controllers\Catlog.php(186): App\Models\Catlog_model->delete_img('13')
 3 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Catlog->delete_img()
 4 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Catlog))
 5 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-01-23 03:29:20 --> Undefined array key "attached_file_name"
in APPPATH\Models\Catlog_model.php on line 514.
 1 APPPATH\Models\Catlog_model.php(514): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "attached_file_name"', 'APPPATH\\Models\\Catlog_model.php', 514)
 2 APPPATH\Controllers\Catlog.php(186): App\Models\Catlog_model->delete_img('13')
 3 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Catlog->delete_img()
 4 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Catlog))
 5 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-01-23 03:30:30 --> Undefined array key "attached_file_name"
in APPPATH\Models\Catlog_model.php on line 514.
 1 APPPATH\Models\Catlog_model.php(514): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "attached_file_name"', 'APPPATH\\Models\\Catlog_model.php', 514)
 2 APPPATH\Controllers\Catlog.php(186): App\Models\Catlog_model->delete_img('12')
 3 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Catlog->delete_img()
 4 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Catlog))
 5 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-01-23 03:31:15 --> Undefined array key "attached_file_name"
in APPPATH\Models\Catlog_model.php on line 514.
 1 APPPATH\Models\Catlog_model.php(514): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "attached_file_name"', 'APPPATH\\Models\\Catlog_model.php', 514)
 2 APPPATH\Controllers\Catlog.php(186): App\Models\Catlog_model->delete_img('12')
 3 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Catlog->delete_img()
 4 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Catlog))
 5 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-01-23 03:39:27 --> Undefined array key "attached_file_name"
in APPPATH\Models\Catlog_model.php on line 513.
 1 APPPATH\Models\Catlog_model.php(513): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "attached_file_name"', 'APPPATH\\Models\\Catlog_model.php', 513)
 2 APPPATH\Controllers\Catlog.php(186): App\Models\Catlog_model->delete_img('12')
 3 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Catlog->delete_img()
 4 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Catlog))
 5 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-01-23 03:40:03 --> unlink(public/assets/productImgs/project2.jpeg): No such file or directory
in APPPATH\Models\Catlog_model.php on line 515.
 1 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'unlink(public/assets/productImgs/project2.jpeg): No such file or directory', 'APPPATH\\Models\\Catlog_model.php', 515)
 2 APPPATH\Models\Catlog_model.php(515): unlink('public/assets/productImgs/project2.jpeg')
 3 APPPATH\Controllers\Catlog.php(186): App\Models\Catlog_model->delete_img('11')
 4 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Catlog->delete_img()
 5 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Catlog))
 6 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-01-23 03:40:54 --> unlink(C:\xampp\htdocs\sumscircle\public/assets/productImgs/project2.jpeg): No such file or directory
in APPPATH\Models\Catlog_model.php on line 515.
 1 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'unlink(C:\\xampp\\htdocs\\sumscircle\\public/assets/productImgs/project2.jpeg): No such file or directory', 'APPPATH\\Models\\Catlog_model.php', 515)
 2 APPPATH\Models\Catlog_model.php(515): unlink('FCPATH\\public/assets/productImgs/project2.jpeg')
 3 APPPATH\Controllers\Catlog.php(186): App\Models\Catlog_model->delete_img('11')
 4 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Catlog->delete_img()
 5 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Catlog))
 6 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-01-23 03:41:49 --> unlink(): http does not allow unlinking
in APPPATH\Models\Catlog_model.php on line 515.
 1 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'unlink(): http does not allow unlinking', 'APPPATH\\Models\\Catlog_model.php', 515)
 2 APPPATH\Models\Catlog_model.php(515): unlink('http://localhost/sumscircle/public/assets/productImgs/project2.jpeg')
 3 APPPATH\Controllers\Catlog.php(186): App\Models\Catlog_model->delete_img('11')
 4 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Catlog->delete_img()
 5 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Catlog))
 6 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-01-23 03:42:55 --> unlink(): http does not allow unlinking
in APPPATH\Models\Catlog_model.php on line 515.
 1 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'unlink(): http does not allow unlinking', 'APPPATH\\Models\\Catlog_model.php', 515)
 2 APPPATH\Models\Catlog_model.php(515): unlink('http://localhost/sumscircle/public/assets/productImgs/project2.jpeg')
 3 APPPATH\Controllers\Catlog.php(186): App\Models\Catlog_model->delete_img('11')
 4 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Catlog->delete_img()
 5 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Catlog))
 6 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-01-23 03:44:33 --> Undefined variable $path2
in APPPATH\Models\Catlog_model.php on line 515.
 1 APPPATH\Models\Catlog_model.php(515): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $path2', 'APPPATH\\Models\\Catlog_model.php', 515)
 2 APPPATH\Controllers\Catlog.php(186): App\Models\Catlog_model->delete_img('11')
 3 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Catlog->delete_img()
 4 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Catlog))
 5 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-01-23 03:44:55 --> Undefined variable $path2
in APPPATH\Models\Catlog_model.php on line 515.
 1 APPPATH\Models\Catlog_model.php(515): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $path2', 'APPPATH\\Models\\Catlog_model.php', 515)
 2 APPPATH\Controllers\Catlog.php(186): App\Models\Catlog_model->delete_img('11')
 3 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Catlog->delete_img()
 4 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Catlog))
 5 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-01-23 03:46:53 --> unlink(): http does not allow unlinking
in APPPATH\Models\Catlog_model.php on line 516.
 1 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'unlink(): http does not allow unlinking', 'APPPATH\\Models\\Catlog_model.php', 516)
 2 APPPATH\Models\Catlog_model.php(516): unlink('http://localhost/sumscircle/public/assets/productImgs/imagesproject2.jpeg')
 3 APPPATH\Controllers\Catlog.php(186): App\Models\Catlog_model->delete_img('11')
 4 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Catlog->delete_img()
 5 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Catlog))
 6 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-01-23 03:48:25 --> unlink(): http does not allow unlinking
in APPPATH\Models\Catlog_model.php on line 516.
 1 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'unlink(): http does not allow unlinking', 'APPPATH\\Models\\Catlog_model.php', 516)
 2 APPPATH\Models\Catlog_model.php(516): unlink('http://localhost/sumscircle/public/assets/images/productImgs/project2.jpeg')
 3 APPPATH\Controllers\Catlog.php(186): App\Models\Catlog_model->delete_img('11')
 4 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Catlog->delete_img()
 5 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Catlog))
 6 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-01-23 04:06:42 --> Trying to access array offset on value of type null
in APPPATH\Views\Catlog\edit_products.php on line 155.
 1 APPPATH\Views\Catlog\edit_products.php(155): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type null', 'APPPATH\\Views\\Catlog\\edit_products.php', 155)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\Catlog\\edit_products.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1134): CodeIgniter\View\View->render('Catlog/edit_products', [], true)
 5 APPPATH\Controllers\Catlog.php(180): view('Catlog/edit_products', [...])
 6 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Catlog->edit_product('2')
 7 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Catlog))
 8 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-01-23 04:17:25 --> Trying to access array offset on value of type null
in APPPATH\Models\Catlog_model.php on line 513.
 1 APPPATH\Models\Catlog_model.php(513): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type null', 'APPPATH\\Models\\Catlog_model.php', 513)
 2 APPPATH\Controllers\Catlog.php(186): App\Models\Catlog_model->delete_img('5')
 3 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Catlog->delete_img()
 4 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Catlog))
 5 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-01-23 04:17:28 --> Trying to access array offset on value of type null
in APPPATH\Models\Catlog_model.php on line 513.
 1 APPPATH\Models\Catlog_model.php(513): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type null', 'APPPATH\\Models\\Catlog_model.php', 513)
 2 APPPATH\Controllers\Catlog.php(186): App\Models\Catlog_model->delete_img('5')
 3 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Catlog->delete_img()
 4 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Catlog))
 5 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
